app/src/main/java/xterminators/spellingbee/cli/CLIController.java

```diff
@@ -3,10 +3,12 @@
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.IOException;
+
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
+import java.util.TreeMap;
```

Comment:
This import is one reason why I would use Map as the type and Tree or Hash as the implementation, as we already have Map for other things, and CLI controller doesn't really care about implementation.